src/applications/combined-debt-portal/combined/sass/partials/_balance-card.scss

```diff
@@ -0,0 +1,25 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.card-balance {
+  font-family: "Bitter";
+  font-size: 24px;
+}
+
+.card-heading {
+  font-family: "Bitter";
+  font-size: 17px;
+  color: $color-gray-dark;
+}
+
+.card-content {
```

Comment:
I believe all these can be replaced with design system utility classes https://design.va.gov/foundation/utilities/flexbox

Reply:
Thank you for pointing this out @timwright12! I was reusing some existing card functionality, so I'll make these corrections and be sure to keep it in mind moving forward!